src/executors/default.yml

```diff
@@ -0,0 +1,7 @@
+description: "The official CircleCI CIMG PHP Docker image."
+parameters:
+  tag:
+    description: "The `cimg/php` Docker image version tag."
+    type: string
```

Comment:
Wondering if it's worth adding a default value to this or not like we do for the `install-php` step.

Reply:
done in https://github.com/CircleCI-Public/php-orb/pull/25/commits/31da694208110e22575ef097caf67105390e5507